plugins/connection/incus.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+# Based on lxd.py (c) 2016, Matt Clay <matt@mystile.com>
+# (c) 2023, Stephane Graber <stgraber@stgraber.org>
+# Copyright (c) 2023 Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    author: Stéphane Graber <stgraber@stgraber.org>
+    connection: incus
+    short_description: Run tasks in Incus instances via the Incus CLI
+    description:
+        - Run commands or put/fetch files to an existing Incus instance using Incus CLI.
+    version_added: "8.2.0"
+    options:
+      remote_addr:
+        description:
+            - Instance identifier
+        default: inventory_hostname
+        vars:
+            - name: ansible_host
+            - name: ansible_incus_host
+      executable:
+        description:
+            - The shell to use for execution inside the instance.
+        default: /bin/sh
+        vars:
+            - name: ansible_executable
+            - name: ansible_incus_executable
+      remote:
+        description:
+            - The name of the Incus remote to use (per "incus remote list").
```

Comment:
```suggestion
            - The name of the Incus remote to use (per C(incus remote list)).
```
(https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#format-macros-within-module-documentation)